src/containers/GameContainer/saveResultLocal.ts

```diff
@@ -0,0 +1,9 @@
+const SCORE = 'maxScore';
+
+export const saveResultLocal = (score: number): void => {
+  const currentMaxScore = localStorage.getItem(SCORE);
+
+  if (score > +currentMaxScore) {
```

Comment:
🧐
Советую писать так, потому что унарный плюс не считывается, если бегло читаешь код. С `Number` работать будет точно так же, но более заметно и явно
```suggestion
  if (score > Number(currentMaxScore)) {
```

Reply:
Вот это супер крутой комментарий, потому что я давно хотел спросить. Вот таких штук типо + и Number в js очень много и всем нравится разное. Например мне и всем у меня в команде больше нравится +. Как решать такие вопросы в группе или это должен решать старший?

Reply:
Такое решать должна команда _вместе_ со старшим, а он не в одно лицо. Когда сверху спускаются директивы насчёт вкусовщины — это фу. Например, последний мой разговор с тимлидом был насчёт деструктуризации пропсов только внутри тела функции (компонент ли это, или просто функция, принимающая параметры объектом). Там явно видны плюсы такого подхода, которые вкусовщиной не объяснишь, но даже в этом случае это не была директива, а нормальная конструктивная дискуссия